### PR TITLE
Task4 - Linked DB to app

### PR DESCRIPTION
**Pull request by willthornhill**

### CODE REVIEW

**Review by EleanorRoberts — APPROVED** [review]
Very nice! Your code is all clean and legible, nicely indented and very consistent! A* from me 💃

File: collection.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <title>Lego Collection</title>
+</head>
+<body>
+<header>
+    <h1>My collection</h1>
+</header>
+
+<div class="main">
+    <div class="item">
+        <?php
```
Comment (Roomsycakes01) [inline collection.php:22]: try to use a request once instead of php in the html

File: collection.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lego Collection</title>
+</head>
+<body>
+<header>
+    <h1>My collection</h1>
+</header>
+
```
Comment (Roomsycakes01) [inline collection.php:19]: whitespace not needed

File: index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lego Collection</title>
+</head>
+<body>
+    <header>
+        <h1>My collection</h1>
+    </header>
+
```
Comment (Roomsycakes01) [inline index.html:11]: whitespace again

File: index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            <h3>Name Placeholder</h3>
+            <img src="placeholder" alt="placeholder image"/>
+            <p>Set Number:</p>
+            <p>Piece Count:</p>
+            <p>Release Year:</p>
+        </div>
+    </div>
+
+</body>
+</html>
```
Comment (Roomsycakes01) [inline index.html:23]: another line at the bottom

File: collection.php [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<?php
+//--QUERY--
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+$query = $db->prepare("SELECT `setName`, `setNumber`, `pieceCount`, `releaseYear` FROM `sets`;");
```
Comment (Roomsycakes01) [inline collection.php:4]: inconsistent quotes

**Review by Roomsycakes01 — COMMENTED** [review]
looks good, just some minor formatting needed and separation of concerns.

**Pull request merged**